samples/openapi3/client/petstore/dart-dio-next/petstore_client_lib_fake/doc/StoreApi.md

```diff
@@ -26,12 +26,12 @@ For valid response try integer IDs with value < 1000. Anything above 1000 or non
 ```dart
 import 'package:openapi/api.dart';
 
-var api_instance = new StoreApi();
```

Comment:
I feel a bit cheated with name of this PR. these changed dont belong here imo.

Reply:
Can you elaborate? The existing samples have been wrong from the first day.

Reply:
These are sample code snippets in markdown files in case you missed that :o